lib/WorkflowMain.groovy

```diff
@@ -10,7 +10,7 @@ class WorkflowMain {
     public static String citation(workflow) {
         return "If you use ${workflow.manifest.name} for your analysis please cite:\n\n" +
             "* The pipeline\n" +
-            "  https://doi.org/10.5281/zenodo.6563578\n\n" +
+            "  https://doi.org/10.5281/zenodo.6563577\n\n" +
```

Comment:
Well earlier I was doing as you are recommending, but found out this is how nf-core is doing it. They add the umbrella citation to their README after the first release. Then in Zenodo on the side you can see the various versions. Yes the latest one opens up automatically but you can select a different version on the side. I have done that for rnaseq in the past.

Using umbrella citation means (1) we do not have to change the links everywhere for each release, and (2) we can submit to Zenodo after release rather than create a dummy entry to reserve a DOI. Let me know your thoughts.

Reply:
Will use umbrella version for pipelines and genome note citation as well.